games-puzzle/sdllopan/sdllopan-10.recipe

```diff
@@ -26,21 +25,20 @@ ARCHITECTURES="x86_gcc2 x86_64"
 
 PROVIDES="
 	sdllopan = $portVersion
-	app:lopan = $portVersion
+	app:Lopan = $portVersion
```

Comment:
Is there any way to set this to "SDL Lopan"?

Reply:
Why? The executable and deskbar entry are called just Lopan and that's how it should be. The end-user doesn't need to know sdl is involved here.

Reply:
I guess a picture tells more then words (check window title) :)
![SDL-Lopan](https://user-images.githubusercontent.com/16057090/68072904-1ec4b480-fd8b-11e9-9e32-cbb0252bfe78.png)

Reply:
app:SDL\ Lopan doesn't work as expected?

Reply:
Need to check :)

Reply:
Looks like it doesn't, maybe I'll just use SDL_Lopan:
```
Warning: failure creating the build package: Parse error in .PackageInfo(23:12) -> invalid character in resolvable name
```

Reply:
I guess we should poke @scottmc for creating the window tittle? ;)